enterprise/app/history/history.tsx

```diff
@@ -117,6 +118,12 @@ export default class HistoryComponent extends React.Component<Props, State> {
         commitSha: this.props.commit || filterParams.commit,
         command: filterParams.command,
         groupId: this.props.user?.selectedGroup?.id,
+        minimumDuration: {
+          seconds: new Long(Number(filterParams.minimumDuration)),
+        },
+        maximumDuration: {
+          seconds: new Long(Number(filterParams.maximumDuration)),
+        },
```

Comment:
nit, you can omit `Long()` here -- our client side RPCs all use regular old numbers currently which is nice because `Long` is sort of a protobufjs-ism and we might want to move away from it at some point.

Reply:
Doesn't work :-(

https://app.buildbuddy.io/invocation/fa15669d-d691-426a-a1e6-4494882aac01